src/Badge/Service/ClientStrategy.php

```diff
@@ -85,6 +99,10 @@ public function getRepositoryPrefix(Repository $repository, string $repoUrl): st
             $repositoryPrefixUrl .= '/'.self::BITBUCKET_REPOSITORY_PREFIX;
         }
 
+        if ($repository->isGitLab()) {
+            return $repoUrl.'/'.self::GITLAB_REPOSITORY_PREFIX;
```

Comment:
we don't have the cs-fixer role that applay the space before and after `.`?

Reply:
Is equal to the `GitHub` method. We can check after the cs-fixer and its rules in a different PR if it's necessary.